baus/datasources.py

```diff
@@ -21,22 +21,25 @@
 # define new settings files- these have been subdivided from the
 # general settings file
 # this is similar to the code for settings in urbansim_defaults
-@orca.injectable('hazards', cache=True)
-def hazards():
-    with open(os.path.join(misc.configs_dir(), "hazards.yaml")) as f:
+@orca.injectable('run_setup', cache=True)
+def run_setup():
+    with open("../run_setup.yaml") as f:
```

Comment:
Part of this PR is moving the inputs, outputs and `run_setup.yaml` external to the repo, stored as a relative path here because I wanted to limit hard coding of these locations to the extent possible. Did I capture your comment? One thing to do is update the repo's main README to reflect this.